### ISSUE TITLE
Failed to install tremor language server

### ISSUE THREAD

**Issue by diru1100** [tremor-rs/tremor-language-server#137]
Error message: 
```
Compiling tremor-language-server v0.10.1
error: failed to run custom build command for `tremor-language-server v0.10.1`

Caused by:
  process didn't exit successfully: `/var/folders/84/zhf78lt50hsghfh5s6g9jmk00000gn/T/cargo-installweU1XI/release/build/tremor-language-server-cf2b09233f27bfff/build-script-build` (exit code: 101)
  --- stdout
  Dumping function docs for tremor-script to: /var/folders/84/zhf78lt50hsghfh5s6g9jmk00000gn/T/cargo-installweU1XI/release/build/tremor-language-server-11042796ebe3999c/out/function_docs.tremor-script.bin
  Detected tremor-script version from cargo lock file: 0.10.1
  Could not find tremor-script src in local cargo registry, so downloading it now...
  Downloading crate `tremor-script==0.10.1` from https://crates.io/api/v1/crates/tremor-script/0.10.1/download

  --- stderr
  thread 'main' panicked at 'there is no reactor running, must be called from the context of a Tokio 1.x runtime', /Users/rohitdandamudi/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.2.0/src/runtime/blocking/pool.rs:85:33
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `tremor-language-server v0.10.1`, intermediate artifacts can be found at `/var/folders/84/zhf78lt50hsghfh5s6g9jmk00000gn/T/cargo-installweU1XI`

Caused by:
  build failed
```

OS: MacOS Mojave
Command: tremor-runtime rohitdandamudi$ cargo install tremor-language-server

**Comment by mfelsche** [comment]
We didnt release a new language-server yet, it seems. Gonna change that today.
Sorry for the inconvenience!

**Comment by mfelsche** [comment]
can you try again? I just released `0.10.2`.

**Comment by diru1100** [comment]
Yup, it works now. 
Thank you!! 😁
